.github/workflows/sec.yml

```
@@ -0,0 +1,20 @@
+
+on:
+  pull_request:
+    types: [opened, synchronize, labeled]
+  push:
+    branches:
+      - main
+      - 'release/*'
+
+jobs:
+  sec:
+    uses: ./.github/workflows/sec-workflow.yml
```

actionlint (1.7.10)

could not read reusable workflow file for "./.github/workflows/sec-workflow.yml": open /home/jailuser/git/.github/workflows/sec-workflow.yml: no such file or directory

(workflow-call)